cmd/troubleshoot/cli/root.go

```diff
@@ -68,7 +68,7 @@ from a server that can be used to assist when troubleshooting a Kubernetes clust
 	cmd.Flags().Bool("redact", true, "enable/disable default redactions")
 	cmd.Flags().Bool("interactive", true, "enable/disable interactive mode")
 	cmd.Flags().Bool("collect-without-permissions", true, "always generate a support bundle, even if it some require additional permissions")
-	cmd.Flags().StringSliceP("selector", "l", []string{"troubleshoot.io/kind=support-bundle"}, "selector to filter on for loading additional support bundle specs found in secrets within the cluster")
+	cmd.Flags().StringSliceP("selector", "l", []string{"troubleshoot.sh/kind=support-bundle"}, "selector to filter on for loading additional support bundle specs found in secrets within the cluster")
```

Comment:
if we're going to deprecate troubleshoot.io, but keep support for it, what should we have as the default here?  I think the new, but want feedback on that given that we want to support the old and the new label.  For one to be default, that means that folks using the other will need to supply it on the CLI, unless we add logic for multiple labels and have this default to both labels.